### DISCUSSION TITLE
[WARN] onetagger_autotag: Shazam failed: Unexpected IO Error: Unable to find the specified path (os error 3) at offset 0 (0x0)

### DISCUSSION THREAD

**Discussion by antscribano** [Marekkon5/onetagger#256] [Q&A]
Hello,
I'm trying to auto tag several files placed in several folders by using an m3u file created with VLC.

While autotagging using Shazam I get the folllowing error:

> 2023-08-14 13:36:32 [WARN] onetagger_autotag: Shazam failed: Unexpected IO Error: Unable to find the specified path (os error 3) at offset 0 (0x0); measured duration up to here: 0ns

Any idea why this is happening?
Thank you

**Comment by Marekkon5** [comment]
Hello, perhaps some of the files that are mentioned in the m3u do not exist, or the paths are invalid (ie. the paths are relative to VLC, but 1t is running in a different directory).

**Reply by antscribano** [reply]
Uhm, sounds possibile. Anyway I also tried by adding absolute path to files, but nothing happens.

What software do you use tu create m3u files?

Thanks

**Reply by Marekkon5** [reply]
Uh I generally don't use M3U and make them by hand when testing, however can you send the .m3u file so we can see if it's ok/valid? Or perhaps one of the files does not exist. Thanks

**Reply by antscribano** [reply]
Here we go: 
[no-title.zip](https://github.com/Marekkon5/onetagger/files/12334527/no-title.zip)

When using relative paths, the file was placed on Desktop. "Musica" is a subfolder of Desktop.

Relative paths had this structure:

#EXTM3U
#EXTINF:496,104-etnoscope_-_pannalized-upe.mp3
Musica/00.%20Altro/History%20of%20Goa%20Trance/Goa%20Head%20Discography%2020cd%20(The%20Best%20Of%20Goa%20Trance%20And%20Psychedelic%20Techno)/goa%20head/volume%2020/cd1/104-etnoscope_-_pannalized-upe.mp3

None of the files is found when using relative or absolute path.

Thank you

**Reply by Marekkon5** [reply]
It seems that VLC URL-encodes local paths, which uh isn't probably standard.
I added a fix in latest commit, you can get the binary from the Actions tab to test it out.

**Reply by antscribano** [reply]
Hello,
I've installed the Win version from the new commit but now, when dragging and dropping the m3u file, this is not loaded on OneTagger.
Can you please check?
Thank you

**Reply by Marekkon5** [reply]
Hello, should be fixed in latest commit, you can get the build from Actions tab to test it. Thanks
